### DISCUSSION TITLE
Release 0.9.14

### DISCUSSION THREAD

**Discussion by HarrySky** [FreeOpcUa/opcua-asyncio#475] [General]
New [tag](https://github.com/FreeOpcUa/opcua-asyncio/releases/tag/0.9.14) is 0.9.14, when actual version is 0.9.13 and neither are uploaded to PyPI :smile:

**Comment by oroulet** [comment]
arrg the release script failed... and now I have merged things on master I do not want to release.... will need to do everything manually

**Comment by oroulet** [comment]
hopefully done

**Reply by HarrySky** [reply]
Yup, 0.9.14 is downloadable from PyPI

**Reply by pbertoni89** [reply]
Hi there, I'm using [0.9.14 from PyPI](https://pypi.org/project/asyncua/0.9.14/) but I'd like to move this dependency from pip to git submodule.
For stability purposes, I'd checkout tag `v0.9.14` first of all, then I will eventually move to more recent versions. As of today, tag `v0.9.14` is still missing from repository, there's [0.9.14](https://github.com/FreeOpcUa/opcua-asyncio/tree/0.9.14) without the `v` prefix. Is this a planned behaviour?

**Reply by oroulet** [reply]
As written over there was an issue when doing that release and some stuff were done by hand. So yes it looks like the v is missing

**Reply by pbertoni89** [reply]
Yep it was just to be sure about that. Thanks a lot
